apps/experiments/models.py

```diff
@@ -891,6 +892,15 @@ def get_trend_data(self) -> tuple[list, list]:
         cache.set(self.trends_cache_key, trend_data, settings.EXPERIMENT_TREND_CACHE_TIMEOUT)
         return trend_data
 
+    def traces_url(self) -> str:
+        """
+        Returns a URL to the traces page, filtered to show only traces for this experiment.
+        """
+        query_params = urllib.parse.urlencode(
+            {"filter_0_column": "experiment", "filter_0_operator": "any of", "filter_0_value": f"[{self.id}]"}
```

Comment:
I want to write a mini utility method or something for these kinds of operations once the refactor PR is in